pkg/ipcmd/ipcmd.go

```diff
@@ -2,23 +2,16 @@
 package ipcmd
 
 import (
+	"fmt"
 	"regexp"
 	"strings"
 
 	"github.com/openshift/openshift-sdn/pkg/exec"
 )
 
-var ipcmdPath string
 var addressRegexp *regexp.Regexp
 
 func init() {
-	var err error
-
-	ipcmdPath, err = exec.LookPath("ip")
```

Comment:
don't we expect 'ip' binary to be present? we do call 'ip link/route ...' in few places. May be for non-redhat network plugin, we can't  expect this binary.
Another option is to set ipcmdPath in NewTransaction() similar to ovs command path.